src/app/locator/qgsinbuiltlocatorfilters.cpp

```diff
@@ -290,12 +321,16 @@ void QgsActiveLayerFeaturesLocatorFilter::prepare( const QString &string, const
   {
     mAttributeAliases.append( layer->attributeDisplayName( idx ) );
   }
+
+  return completionList;
 }
 
 void QgsActiveLayerFeaturesLocatorFilter::fetchResults( const QString &string, const QgsLocatorContext &, QgsFeedback *feedback )
 {
   int found = 0;
   QgsFeature f;
+  QString searchString = string;
+  fieldRestriction( searchString );
```

Comment:
Should that use the return value somewhere?

Reply:
nope, the return value is only used in the other call.